keyboards/mt980/info.json

```
@@ -0,0 +1,117 @@
+        {"label":"KA1", "x":16.5, "y":1.5},
+        {"label":"KA0", "x":17.5, "y":1.5},
+        {"label":"K64", "x":18.5, "y":1.5},
+        {"label":"K30", "x":0, "y":2.5, "w":1.5},
+        {"label":"K31", "x":1.5, "y":2.5},
+        {"label":"K32", "x":2.5, "y":2.5},
+        {"label":"K33", "x":3.5, "y":2.5},
+        {"label":"K34", "x":4.5, "y":2.5},
+        {"label":"K35", "x":5.5, "y":2.5},
+        {"label":"K36", "x":6.5, "y":2.5},
+        {"label":"K37", "x":7.5, "y":2.5},
+        {"label":"K38", "x":8.5, "y":2.5},
+        {"label":"K98", "x":9.5, "y":2.5},
+        {"label":"K97", "x":10.5, "y":2.5},
+        {"label":"K95", "x":11.5, "y":2.5},
+        {"label":"K94", "x":12.5, "y":2.5},
+        {"label":"K84", "x":13.5, "y":2.5, "w":1.5},
+        {"label":"K96", "x":15.5, "y":2.5},
+        {"label":"K92", "x":16.5, "y":2.5},
+        {"label":"K91", "x":17.5, "y":2.5},
+        {"label":"K90", "x":18.5, "y":2.5},
+        {"label":"K20", "x":0, "y":3.5, "w":1.75},
+        {"label":"K21", "x":1.75, "y":3.5},
+        {"label":"K22", "x":2.75, "y":3.5},
+        {"label":"K23", "x":3.75, "y":3.5},
+        {"label":"K24", "x":4.75, "y":3.5},
+        {"label":"K25", "x":5.75, "y":3.5},
+        {"label":"K26", "x":6.75, "y":3.5},
+        {"label":"K27", "x":7.75, "y":3.5},
+        {"label":"K28", "x":8.75, "y":3.5},
+        {"label":"K88", "x":9.75, "y":3.5},
+        {"label":"K87", "x":10.75, "y":3.5},
+        {"label":"K85", "x":11.75, "y":3.5},
+        {"label":"K93", "x":12.75, "y":3.5, "w":2.25},
+        {"label":"K86", "x":15.5, "y":3.5},
+        {"label":"K82", "x":16.5, "y":3.5},
+        {"label":"K81", "x":17.5, "y":3.5},
+        {"label":"K80", "x":18.5, "y":3.5},
+        {"label":"K10", "x":0, "y":4.5, "w":1.25},
+        {"label":"K11", "x":1.25, "y":4.5},
+        {"label":"K12", "x":2.25, "y":4.5},
+        {"label":"K13", "x":3.25, "y":4.5},
+        {"label":"K14", "x":4.25, "y":4.5},
+        {"label":"K15", "x":5.25, "y":4.5},
+        {"label":"K16", "x":6.25, "y":4.5},
+        {"label":"K17", "x":7.25, "y":4.5},
+        {"label":"K18", "x":8.25, "y":4.5},
+        {"label":"K78", "x":9.25, "y":4.5},
+        {"label":"K77", "x":10.25, "y":4.5},
+        {"label":"K75", "x":11.25, "y":4.5},
+        {"label":"K74", "x":12.25, "y":4.5, "w":1.75},
+        {"label":"K76", "x":15.5, "y":4.5},
+        {"label":"K72", "x":16.5, "y":4.5},
+        {"label":"K71", "x":17.5, "y":4.5},
+        {"label":"K70", "x":18.5, "y":4.5},
+        {"label":"K73", "x":14.25, "y":4.75},
+        {"label":"K00", "x":0, "y":5.5, "w":1.25},
+        {"label":"K01", "x":1.25, "y":5.5},
+        {"label":"K02", "x":2.25, "y":5.5, "w":1.25},
+        {"label":"K06", "x":3.5, "y":5.5, "w":6.25},
+        {"label":"K08", "x":9.75, "y":5.5},
+        {"label":"K07", "x":10.75, "y":5.5},
+        {"label":"K05", "x":11.75, "y":5.5, "w":1.25},
+        {"label":"K62", "x":16.5, "y":5.5},
```

K04, K03 and K66 should be before K62.

```suggestion
        {"label":"K04", "x":13.25, "y":5.75},
        {"label":"K03", "x":14.25, "y":5.75},
        {"label":"K66", "x":15.25, "y":5.75},
        {"label":"K62", "x":16.5, "y":5.5},
```